scripts/tomcat/CopyWebAppsToTomcat.sh

```diff
@@ -21,10 +21,15 @@ sudo -u $USER -g tomcat rm -rf $TOMCAT8_HOME/webapps/hoot-services/
 
 sudo -u $USER -g tomcat cp `ls -t hoot-services/target/hoot-services-*.war | sed -n 1p` $TOMCAT8_HOME/webapps/hoot-services.war
 
+# Copy hootenanny-ui web files
 sudo -u $USER -g tomcat rm -rf $TOMCAT8_HOME/webapps/hootenanny-id/
 sudo -u $USER -g tomcat cp -R hoot-ui/dist $TOMCAT8_HOME/webapps/hootenanny-id
 sudo -u $USER -g tomcat mkdir -p $TOMCAT8_HOME/webapps/hootenanny-id/data
 sudo -u $USER -g tomcat cp hoot-ui/data/osm-plus-taginfo.csv $TOMCAT8_HOME/webapps/hootenanny-id/data
 sudo -u $USER -g tomcat cp hoot-ui/data/tdsv61_field_values.json $TOMCAT8_HOME/webapps/hootenanny-id/data
 
+#Copy hootenanny-ui2x we files
+sudo -u $USER -g tomcat rm -rf $TOMCAT8_HOME/webapps/hootenanny-id2x/
+sudo -u $USER -g tomcat cp -R hoot-ui-2x/dist $TOMCAT8_HOME/webapps/hootenanny-id2x
+
```

Comment:
@putipong Did the `osm-plus-taginfo.csv` and 'tdsv61_fields_values.json` files get included into the build in the `hoot2x` branch?  @sputnam If not we'll probably want to include those files in the 2x deploy so that translation assistant will still function.

Reply:
@brianhatchl they get put in the correct place by the hoot-ui-2x build.